mappings/ab0dec21aa7d939ad124ffaf95faf8ed72b1018ff72fffd8f9974907474f4b4559.json

```diff
@@ -0,0 +1,64 @@
+        "signatures": [
+            {
+                "signature": "bdabf4114a84a6f0ccf7747c9ac43bf134df09e7d448f21e51b73233e3f753811b90d5de76babe466b0b5ac40058d8e4c23ceb55e5da5706f0673f09a652b303",
+                "publicKey": "0a39954288c8bcec75c7ec374ad6dd2071bdb5df854a920572b37b4a22fa3b6a"
+            }
+        ]
+    },
+    "description": {
+        "sequenceNumber": 0,
+        "value": "GOKEY is the World's first cryptocurrency to dynamically adjust its digital scarcity at rates corresponding to changes in internationally recognized real estate price indices, and is the primary governance, payment and utility token of the GoKey real estate financial ecosystem.",
+        "signatures": [
+            {
+                "signature": "1147c861f5ac5e7c9ba9432664a960af87232ac9e88e3b4267da46ecabb8ef0bd150a05cbb2d888f83ee5b668a8683a9f615eb720ee97faa3d6a5dc9d85aa806",
```

Comment:
@craigimmel,

signature for the description seems to be wrong. Can you please resign the file and update the PR?